gm4_display_frames/data/gm4_display_frames/functions/main.mcfunction

```diff
@@ -0,0 +1,2 @@
+execute at @a[gamemode=!spectator] as @e[type=item_frame,tag=gm4_df_invis_frame,distance=..16] run function gm4_display_frames:process

```

Comment:
Keep in mind that this may call `process` multiple times on a single frame, if that frame is in range of multiple players.